packages/apps-config/src/endpoints/testing.ts

```diff
@@ -912,6 +912,18 @@ export const testChains: Omit<EndpointOption, 'teleport'>[] = [
       logo: nodesSubspacePNG
     }
   },
+  {
+    info: 'subspace-gemini-3h',
+    providers: {
+      US: 'wss://rpc-0.gemini-3h.subspace.network/ws',
+      US2: 'wss://rpc-1.gemini-3h.subspace.network/ws'
```

Comment:
@DaMandal0rian are any of these actually in the US?

Reply:
@nazar-pc this is correct, they are both in the US